### PR TITLE
Python3

### PR DESCRIPTION
**Pull request by braunfuss**
Bugfixes to load of models and plotting

### CODE REVIEW

File: src/plotting.py [hunk 1]
```diff
@@ -1707,7 +1706,7 @@ def draw_correlation_hist(problem, plot_options):
         varnames = problem.hypernames
     else:
         sc = problem.config.sampler_config
-        varnames = problem.varnames + problem.hypernames + ['like']
+        varnames = list(problem.varnames) + problem.hypernames + ['like']
```
Comment (hvasbath) [inline src/plotting.py]: let us fix this in the problems.varnames property

File: src/models/geodetic.py [hunk 2]
```diff
@@ -322,8 +322,9 @@ def point2sources(self, point):
                 tpoint.pop(hyper)
 
         source_params = self.sources[0].keys()
+        source_params_arr= num.array(list(tpoint.keys()))
```
Comment (hvasbath) [inline src/models/geodetic.py]: why array and not list?

File: src/backend.py [hunk 3]
```diff
@@ -715,13 +715,13 @@ def load_multitrace(dirname, varnames=None, chains=None):
     if not istransd(varnames)[0]:
         logger.info('Loading multitrace from %s' % dirname)
         if chains is None:
-            files = glob(os.path.join(dirname, 'chain-*.csv'))
+            files = glob(os.path.join(dirname, 'chain*.csv'))
```
Comment (hvasbath) [inline src/backend.py]: whats the issue with the minus? this change like this will brake sth else, we need the minus

File: src/plotting.py (hunk 1 above)
Comment (braunfuss) [reply src/plotting.py]: ok sure, i was not sure if this would pop out as a problem somewhere else than,

File: src/models/geodetic.py (hunk 2 above)
Comment (braunfuss) [reply src/models/geodetic.py]: the iteration in the next lines will else not work, as the space to iterate over will change if using a dict

File: src/backend.py (hunk 3 above)
Comment (braunfuss) [reply src/backend.py]: else he tries to load in chain--* 
I am also confused where the addtional - comes from, so i just removed it here. Load in now works with this.

**Pull request merged**